turborepo-tests/integration/fixtures/turbo-configs/interruptible-but-not-persistent.json

```diff
@@ -10,7 +10,7 @@
     "build": {
       "env": [
         "NODE_ENV",
-        "$FOOBAR"
+        "FOOBAR"
```

Comment:
We changed validation order (both are still checked) and this `turbo.json` was invalid in 2 ways. Previously we were depending on the ordering of validations to see the expected error message around this task being interruptible, but not persistent.